source/systems/cloud_user_guide.rst

```diff
@@ -117,6 +117,302 @@ This includes data stored in a CSP public, free to access repositories, like
 the `NOAA Open Data Dissemination (NODD) <NOAA NODD_>`_ program.
 
 
+Using Parallel Works
+====================
+
+Before you Begin
+----------------
+
+NOAA Cloud Computing uses the Parallel Works ACTIVATE platform. ACTIVATE allows
+users to manage their cloud computing resources across Amazon Web Services
+(AWS), Google Compute Platform (GCP), and Microsoft Azure Cloud Computing
+Services (Azure).  Users access ACTIVATE via the customized NOAA RDHPCS Portal.
+
+.. note::
+
+  The `Parallel Works User guide <https://parallelworks.com/docs>`_ provides comprehensive
+  information for using the ACTIVATE control plane.
+
+The certified browser for Parallel Works is Google Chrome. To use the ACTIVATE
+platform, you must have a NOAA user account and password, and a valid RSA
+token. Click the links for instructions for :ref:`applying_for_user_account`
+and obtaining :ref:`rsa-token`.
+
+You must also be assigned to a Cloud project account.  To join a Cloud project,
+first request the project name from your PI, TL, or Portfolio Manager. Then use
+the AIM tool to `request access to that project
+<https://docs.rdhpcs.noaa.gov/accounts/accounts_and_projects.html#request-access-to-rdhpcs-projects>`_.
+
+NOAA’s Parallel Works Portal
```

Comment:
There's already a section header with this title above on the page. This data should be consolidated, or this should be renamed, to avoid confusion.